python/cuml/dask/ensemble/base.py

```diff
@@ -20,40 +38,32 @@ def _create_the_model(self, model_func, **kwargs):
 
         n_est_per_worker = math.floor(self.n_estimators / n_workers)
 
-        for i in range(n_workers):
-            self.n_estimators_per_worker.append(n_est_per_worker)
-
+        self.n_estimators_per_worker = \
+            [n_est_per_worker for i in range(n_workers)]
         remaining_est = self.n_estimators - (n_est_per_worker * n_workers)
 
         for i in range(remaining_est):
             self.n_estimators_per_worker[i] = (
                 self.n_estimators_per_worker[i] + 1
             )
-        seeds = list()
-        seeds.append(0)
+        seeds = [0]

```

Comment:
I'm probably just misunderstanding, but why does seeds always contain 0 for one element? Should a user be able to pass in the base seed?